tests/integration/cots_test.py

```diff
@@ -1689,25 +1689,25 @@ def test_cots_purge_trip_and_rt(mock_rabbitmq):
         # Check there's really something before purge
         assert RealTimeUpdate.query.count() == 1
 
-        config = {
-            "contributor": COTS_CONTRIBUTOR_ID,
-            "nb_days_to_keep": DEFAULT_DAYS_TO_KEEP_TRIP_UPDATE,
-        }
+        contributor = Contributor(COTS_CONTRIBUTOR_ID, "", "")
+        contributor.nb_days_to_keep_trip_update = DEFAULT_DAYS_TO_KEEP_TRIP_UPDATE
         tu = VehicleJourney.query.first()
         tu.start_timestamp = date.today() - timedelta(days=DEFAULT_DAYS_TO_KEEP_TRIP_UPDATE + 1)
-        purge_trip_update(config)
+        tu_purge = TripUpdatePurge()
+        tu_purge.do_purge(contributor)
```

Comment:
It's even clearer to me like this (interface may change, though):
```suggestion
        TripUpdatePurge().do_purge(contributor)
```

Reply:
Done in d1fd57c.